### PR TITLE
refactor: handle images for mayastor and bolt

### PR DESCRIPTION
**Pull request by blaisedias**
Create install-images for bolt in the registry under datacore/
Fix the fallout from that change.

Make re-tag-images more portable.
As a bonus this should just work correctly with older images
which have different image names

### CODE REVIEW

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Review by rohan2794 — APPROVED** [review]
LGTM

**Comment by blaisedias** [comment]
bors try

**Comment by blaisedias** [comment]
bors try

File: scripts/re-tag-images.sh [hunk 1]
```diff
@@ -85,22 +83,37 @@ if [ -z "$ALIAS_TAG" ] ; then
...
-  input_image="${REGISTRY}/$regroot/${name}:${SRC_TAG}"
+  input_image="${REGISTRY}/$registry_subdir/${name}:${SRC_TAG}"
 
   docker pull "${input_image}"
 
   if [ "$DESTINATION_REGISTRY" == "dockerhub" ]; then
-    output_image="$regroot/${name}:${ALIAS_TAG}"
+    output_image="$registry_subdir/${name}:${ALIAS_TAG}"
     # do not upload install-images to dockerhub
     if [ "$name" == "install-images" ]; then
```
Comment (chriswldenyer) [inline scripts/re-tag-images.sh:112]: We still need to re-tag install-images for the local registry ? It won't be found in helm charts.

**Review by chriswldenyer — APPROVED** [review]
LGTM, except that I don't think we are now re-tagging install-images itself IIUIC.

File: scripts/re-tag-images.sh (hunk 1 above)
Comment (blaisedias) [reply scripts/re-tag-images.sh:116]: Yes we do. Fixed

**Comment by blaisedias** [comment]
bors merge

**Pull request merged**